docs/scaffold.md

```diff
@@ -0,0 +1,28 @@
+# Start project
+
+## Overview
+
+To see how easy is using Bigflow you can use `start-project` which creates example standalone project with [standard structure](https://github.com/allegro/bigflow/blob/master/docs/project_setup_and_build.md#project-structure).
+This project contains Apache Beam workflow and BigQuery workflow.
+You will be able to deploy those workflows in your GCP project providing informations like project_id/bucket etc.
+
+Beam workflow example counts letters in array and then saves this information to file in your bucket.
```

Comment:
The Beam workflow example counts letters in an array and then saves this information to a file in your bucket.